styles.css

```diff
@@ -415,5 +419,216 @@
 }
 
 .highlight:hover {
-  background-color: #fff !important;
+  background-color: rgb(6, 6, 41) !important;
+}
+
+@media (min-width: 600px) {
+  .desktop-header {
+    background-image: url("/images/Header-llustration-desktop.svg");
+    background-repeat: no-repeat;
+    left: 27px;
+    top: -398px;
+  }

```

Comment:
- [ ] Kindly use media queries for the Desktop, the value for the min-width property should be in the range of 768px or wider) for the proper size.

Reply:
Wow I had overlooked that thank you 🙏 I adjusted accordingly.

![image](https://user-images.githubusercontent.com/59999191/165788480-78154da4-07e8-466a-970e-d9765685baa5.png)